src/index.js

```diff
@@ -1,3 +1,10 @@
+const propertiesToConvert = {
```

Comment:
Turns out that CSSJanus doesn't change everything ending in `left` or `right`. Instead it has all the relevant properties hard-coded (in regexes). I thought this would be a good thing to do for us as well.